src/fontra/views/editor/editor.js

```diff
@@ -714,6 +722,26 @@ export class EditorController {
     );
   }
 
+  async doSelectAllNone(selectNone) {
+    const positionedGlyph =
+      await this.sceneController.sceneModel.getSelectedPositionedGlyph();
+
+    if (positionedGlyph) {
+      const glyphPath = positionedGlyph.glyph.path;
+      const allPoints = new Set();
+
+      for (const [pointIndex] of glyphPath.pointTypes.entries()) {
```

Comment:
Do this only if `!selectNone`

Reply:
(Good use of `.entries()`! I didn't think of that.)

Reply:
In the same `!selectNone` block, also add a loop that adds all `component/${componentIndex}` indices.